src/views/OppiaGettingStarted.vue

```diff
@@ -0,0 +1,138 @@
+
+# Get Started Learning the Technologies Oppia Uses
+
+If you aren't familiar with the following technologies, we strongly encourage you to check out some of the listed resources to develop a basic understanding. We don't expect anyone to become an expert in these technologies before starting, but having seen the basics will help you get started. We suggest spending no more than a couple hours learning each new topic, and many will be much faster!
+
+* Git: Git is a tool for managing many different versions of a file and the history of changes that have been made to it. It is commonly used by software developers to track different versions of their code. Our very own Drew Gregory created a tutorial to introduce you to using git, which you can find [here](https://stanford-code-the-change-guides.readthedocs.io/en/latest/guide_git.html).
+* HTML: HTML is a language used across the web to style the text on webpages. Further, it often defines the basic framework within which the rest of a site is constructed. Mozilla has a [guide](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML) you can check out.
+* Javascript: Javascript implements the logic in many websites. Mozilla has a [guide](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics) you can use, but don't worry about learning much more than the very elementary here because Oppia uses AngularJS instead of using plain Javascript. However, knowing a bit of Javascript will make it much easier to learn AngularJS.
+* AngularJS (v1): AngularJS is a framework that makes writing Javascript easier. We recommend the [official tutorial](https://docs.angularjs.org/tutorial/index), and Oppia also recommends [this video](https://www.youtube.com/watch?v=nO1ROKMjPqI&list=PLvZkOAgBYrsS_ugyamsNpCgLSmtIXZGiz). Oppia's frontend is written in AngularJS, and there are more projects available in the frontend than the backend. Thus, we highly recommend developing your AngularJS knowledge, even if you already know Oppia's backend languages.
+
+Backend (python) and developer tooling (bash) projects are available, but there aren't as many. This is why we recommend learning AngularJS.
+
+# Starter Issues
+
+At this point, you should be able to run the Oppia site locally on your computer, and you should have a basic understanding of Git, HTML, and AngularJS. For starter issues, check out [our trello](https://trello.com/b/ScAy6Mu5/oppia). Let us know what you'd like to work on, and we'll reserve that issue for you. If nothing there strikes your fancy, take a look at the pinned issues at the top of the [Oppia issues page](https://github.com/oppia/oppia/issues).
+
+        </vue-markdown>
+
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import VueMarkdown from 'vue-markdown'
+
+export default {
+  data: function () {
+    return {}
+  },
+  computed: {
+  },
+  methods: {
+  },
+  name: 'ProjectCriteria',
```

Comment:
again, just make sure you have a unique name here.

Reply:
Thanks for the catch! Resolved by a921da53b77179265a6989bac13a480e7d0702eb